android/app/build.gradle

```diff
@@ -221,7 +217,7 @@ dependencies {
 
   /* UNCOMMENT WHEN DISTRIBUTING
   api 'org.webkit:android-jsc:r245459' // needs to be before react-native
-  api 'com.facebook.react:react-native:33.0.0'
+  api 'com.facebook.react:react-native:+'
```

Comment:
Shouldn't it be 🤔 
```suggestion
  api 'com.facebook.react:react-native:34.0.0'
```

Reply:
I've talked to him about it. 
He suggested changing it to `+`.

Reply:
@bbarthec why 34? 😄 it's actually our fault that we haven't been updating this when droping previous SDKs and it's just a coincidence it worked.